src/AppKernel.php

```diff
@@ -208,4 +214,35 @@ public function setRoutingFile($routingFile)
     {
         $this->routingFile = $routingFile;
     }
+
+    public function handleOptions(array $options)
```

Comment:
This is a new feature you snuck in. Why is it better to have an array of values rather than calling the underlying methods?

Reply:
😄 Yeah.

I liked the idea of the options in the `KernelTestCase`, you are right, the methods are better.

The reason why I introduced this is: You only have the possibility to change the config by override the `createKernel` this limits it to a configuration per file and not per test. (If you don't want to boot it yourself and just want to use `static:bootKernel`.)

By just call the `handleOptions` in the `createKernel` you can actually passthrough everything before the kernel is booted and the container is compiled.

Another Idea, that just came in my mind, we could just create a option like `config` which must be a callable:

```
static::bootKernel(['config' => static function(AppKernel $kernel){
    $kernel->setRoutingFile(__DIR__.'/Fixtures/routes.yml');
}]);
```

Then we can benefit from the methods and save some lines, cause the `KernelTestCase` makes the booting process.